.jenkins/pytorch/test.sh

```diff
@@ -42,6 +42,9 @@ if [[ "$BUILD_ENVIRONMENT" != *ppc64le* ]]; then
   # TODO: move this to Docker
   pip install -q hypothesis --user
 
+  # TODO: move this to Docker
+  pip install -q tb-nightly --user
```

Comment:
`tb-nightly` doesn't have the required code yet. It will once https://github.com/tensorflow/tensorboard/pull/2065 lands.

Reply:
I uploaded two files in my web space:
http://m100.nthu.edu.tw/~s100062805/tensorboard/tensorboard-1.14.0a0-py3-none-any.whl
http://m100.nthu.edu.tw/~s100062805/tensorboard/tensorboard-1.14.0a0-py2-none-any.whl